### ISSUE TITLE
Some Feature/Enhancments Request

### ISSUE THREAD

**Issue by max-rh** [mk-5/fjira#109]
After using fjira for a while; I would like to suggest some feature requests that would make fjira even more usable:

- list tickets for a user from ALL projects
- faster scrolling (pg-up & pg-down like in k9s)
- multiple ticket status filtering (filter all ticket that are open & in-progress for example)
- save view for startup (for ex. once I open fjira; i would like to jump directly to a certain view); most probably a save shortcut in a certain view that saves that query in fjira config file and make it load it once it starts
- fuzzy search doesnt work on partial ticket IDs

probably most of these can be achieved with jql and some filters but it would be awesome to get them from within the app.

**Comment by mk-5** [comment]
@max-rh thanks for that features request! :) 

I think all of those features sounds really reasonable. I will try to bring them to fjira.

Question about that one:  `fuzzy search doesnt work on partial ticket IDs` 

what do you mean, partial ticket ids?

**Comment by max-rh** [comment]
@mk-5 Thanks for reply.
I faced an issue where i try to filter by Ticket IDs, lets say I have Tickets under the IDs ILT-17893 to ILT-18000 when I type **ILT-17** nothing appears until i type the full ticket ID.

**Comment by mk-5** [comment]
> @mk-5 Thanks for reply. I faced an issue where i try to filter by Ticket IDs, lets say I have Tickets under the IDs ILT-17893 to ILT-18000 when I type **ILT-17** nothing appears until i type the full ticket ID.

yep - that make sense. The reason is that filter by Ticket ID has no "contains" operator in JQL (jira query langauge). I will check what I can do here

**Comment by mk-5** [comment]
fix for "ALL" issues search is on a way. It was implemented, but apparently it was broken at some point. In order to search within all projects -> just select "All" in projects selection. 

One downside of All projects search is that there is no way to have "status" or "assignee" filters. In order to fetch possible statuses, or assignes you need to have a concrete project id.

**Comment by mk-5** [comment]
pagination using PageUp/PageDown has been added in version 1.3.0

**Comment by max-rh** [comment]
@mk-5 , Thanks for the update; I can confirm the pageup/pagedown works, but the ALL project view doesn't show all tickets under all tickets; I understand that filters and assignee doesn't works, but it should show all assignee and all tickets in all projects, right ?

Plus i noticed that `fjira filters` command doesnt work, it show 404 (this could be due to the same issue with self hosted jira as in #95 )

**Comment by mk-5** [comment]
@max-rh yep it could be the same, I will fix it right away

about the first question - yes, it should work in a way that you described. I will have a look. I didn't test it on "Jira Server" instance.

**Comment by mk-5** [comment]
@max-rh I tried it - and "All" projects search works fine for Jira Server instances. 
When you look into Jira Cloud REST Api docs - https://docs.atlassian.com/software/jira/docs/api/REST/9.2.1/#api/2/search-search

you can read that `maxResults` parameter is controller by server property:

![image](https://github.com/mk-5/fjira/assets/31850900/3acdad87-b428-45cb-85b4-e185db8ffe34)

My guess is that in your instance it's limited to some value, and you are just getting the first "N" jira issues from the list, and they are all from the same project. It should get changed as soon as you type something

**Comment by max-rh** [comment]
I can confirm the PageUp/Down is working now, pretty nice.
And the filters issues is fixed (this one was a life saver, Thanks).

as for the ALL filter, the behavior is weird, i am only getting a subset of the ticket with each filter when i start typing, especially when i filter by username, sometime i get 5 ticket for a certain user, sometime i get 20 for another, so I cant see a patter here.

But tbh, the fitlers fix solves this issue cause i am just building filters for what i need.

**Comment by mk-5** [comment]
@max-rh good to hear. I will try to investigate that weird search behaviour. I has quickly checked it, and at least for my test projects - it was working just fine.

**Comment by gomoz** [comment]
Hello,
PageUp/Down is not working when scrolling through comments on cases.
Some cases have alot of comments, takes some time to go to the buttom.
Is this something that can be applied inside a case also?
